_posts/en/newsletters/2022-10-26-newsletter.md

```diff
@@ -0,0 +1,270 @@
+---
+title: 'Bitcoin Optech Newsletter #223'
+permalink: /en/newsletters/2022/10/26/
+name: 2022-10-26-newsletter
+slug: 2022-10-26-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+- **Continued discussion about full RBF:** in [last week's
+  newsletter][news222 rbf], we summarized a discussion on the
+  Bitcoin-Dev mailing list about the inclusion of a new `mempoolfullrbf`
+  option that could create problems for a several businesses which
+  accept transactions with zero confirmations ("zeroconf") as final
```

Comment:
nit: it says "zeroconf" here, but "zero conf" with a space is used for the rest of the document.